eth_abi/utils/parsing.py

```diff
@@ -1,48 +1,6 @@
 """
 This module provided for backwards compatibility.
 """
-from eth_abi.grammar import (
-    TupleType,
```

Comment:
I must have missed this when it got merged, but worth pointing out now.

The `./project/utils` module in our repos has some conventions that help keep those modules well isolated.

- everything in `utils` is private API
- anything in `utils` may not import from outside of utils.

If you have to break the import rule which occasionally happens, the imports should happen inline in the function bodies to prevent circular imports from happening.

Reply:
Nice.  I'll keep that in mind from here on out.